src/fs/filesystem.ts

```diff
@@ -85,27 +93,6 @@ export class FileSystem {
     return FileSystem.fromCID(cid, opts)
   }
 
-  /**
-   * Upgrade public IPFS folder to FileSystem
```

Comment:
I'm removing this for now. It doesn't currently work with HeaderTrees & there were some questions on if we'd even want to offer this functionality.

Reply:
Did we end up devising a way to get content of a CID into the private part of the filesystem?

Reply:
Not yet. We'll just have to do that manually. The code that was there hasn't worked since we upgraded to `HeaderTree`s. If a user wants to upgrade some IPLD content to FLOOFS, I think we'll need to recursively walk that content and re-add it as FLOOFS objs